### DISCUSSION TITLE
Traceable option for default sliderbody display

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#29610] [Ideas]
Traceable is not only removes hitcircles and sliderheads, it also changes sliderbodies:
- It changes it's color to combo color (instead of custom skin color)
- It makes it fully transparent, so no your usual slider gradient

Some people find this annoying. So if TC players would like to have this customization - it should be added
Related PR - https://github.com/ppy/osu/pull/29600

Additional note - enabling this customization makes TC very slightly easier, as approach circle will have different color from sliderbody, increasing visibility, so probably it means that customization should have slightly lower pp/score bonus. But I'm NOT SURE about, needs discussion.

**Comment by koopafou** [comment]
The point of the mod is to rely on approach circles, so it makes sense that approach circles have the best visibility. I wouldn't recommend talking about reducing pp over this, otherwise you're opening a can of worm about which default skin is easier to play osu with (even if we ignore custom skins), and balance the game around other things like background dim, the presence of hit lighting and other such customizable appearance features.

I will also point out it relates to some issues I raised in #28575 , and aside from the slider body changes, add that circles should reach max opacity earlier, I believe it's on point with the visibility contrast and as a bonus you can limit the skinning hacks when players adjust gamma depending on AR.

Regarding the PR itself, my personal opinion is to keep the inner slider body completely transparent to stay in the visual theme of the mod that we don't really have "full" objects.

**Reply by Givikap120** [reply]
The approach circle opacity is the same as NM in lazer and NM in stable, don't see why it should change in TC
